scripts/contentscript.js

```diff
@@ -16,64 +16,133 @@
 // You should have received a copy of the GNU General Public License
 // along with Foobar.  If not, see <http://www.gnu.org/licenses/>.
 
-/*
- *same with $(document).ready(function(){ });
- *Got it from http://youmightnotneedjquery.com/ 
- */
-function ready(fn) {
```

Comment:
A, content script on browser are run when document is ready

https://developer.chrome.com/extensions/content_scripts

> Note: With "document_idle", content scripts may not necessarily receive the window.onload event, because they may run after it has already fired. In most cases, listening for the onload event is unnecessary for content scripts running at "document_idle" because they are guaranteed to run after the DOM is complete. If your script definitely needs to run after window.onload, you can check if onload has already fired by using the document.readyState property.